bciers/apps/reporting/src/app/components/finalReview/formCustomization/FinalReviewStringField.tsx

```diff
@@ -1,5 +1,40 @@
+import { NumberField } from "@base-ui-components/react/number-field";
 import { FieldProps } from "@rjsf/utils";
 
 export default function FinalReviewStringField(props: FieldProps) {
+  const numberStyles = {
+    font: "inherit",
+    width: "100%",
+    border: "none",
+    backgroundColor: "transparent",
+    color: "inherit",
+    padding: 0,
+  };
+
+  if (props.schema.type === "number") {
+    return (
+      <NumberField.Root
+        id={props.id}
+        name={props.name}
+        disabled
+        value={props.formData ? Number(props.formData) : props.formData}
+        format={{
+          // set the fraction digits based on how many decimal places that value returned from the API has. Lat/long can have up to 8, and some calculated values can have up to 4.
+          maximumFractionDigits: props.formData
+            ? props.formData.toString().split(".")[1]?.length
+            : 4,
```

Comment:
I think this overrides the 4 digits as soon as we have a decimal point in the number, so we might start seeing numbers like 123.00000000000000456 if they ever happen.
For lat/lon we should probably have a separate widget

Reply:
Actually, max decimals should be purely a formatting thing, so that error must be about something else, will investigate